content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,117 @@
+New message format is backward compatible and enabled by default.
+
+Format of structured logs:
+```
+<klog header> "<message>" <key1>="<value1>" <key2>="<value2>" ...
+```
+
+Example:
+```
+I1025 00:15:15.525108       1 controller_utils.go:116] "Pod status updated" pod="kube-system/kubedns" status="ready"
+```
+
+
+### JSON log format
+
+The `--logging-format=json` flag changes format of logs from components to JSON format.
+
+{{<warning >}}
+JSON output does not support many standard klog flags. The [command line tools reference] lists unsupported klog flags.
+
+Not all logs are guaranteed to be written in JSON format (for example, during process start). If you intend to parse logs, make sure you can handle logs in non-JSON formats.
+
+Structured logging is an alpha feature. Field names and JSON serialization are subject to change.
+{{< /warning >}}
+
+Example of JSON log format (pretty printed):
+```json
+{
+   "ts": 1580306777.04728,
+   "v": 4,
+   "msg": "Pod status updated",
+   "pod":{
+      "name": "nginx-1",
+      "namespace": "default"
+   },
+   "status": "ready"
+}
+```
+
+Keys with special meaning:
+* `ts` - timestamp as Unix time (required, float)
+* `v` - verbosity (required, int, default 0)
+* `err` - error string (optional, string)
+* `msg` - message (required, string)
+
+
+List of components currently supporting JSON format:
+* {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+* {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}}
+* {{< glossary_tooltip term_id="kube-scheduler" text="kube-scheduler" >}}
+* {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}
+
+### Log verbosity level
+
+The `-v` flag controls log verbosity. Increasing the value increases the number of logged events. Decreasing the value decreases the number of logged events.
+Increasing verbosity settings logs increasingly less severe events. A verbosity setting of 0 logs only critical events.
+
+### Log location
+
+There are two types of system components: those that run in a container and those
+that do not run in a container. For example:
+
+* The Kubernetes scheduler and kube-proxy run in a container.
+* The kubelet and container runtime, for example Docker, do not run in containers.
+
+On machines with systemd, the kubelet and container runtime write to journald.
+Otherwise, they write to `.log` files in the `/var/log` directory.
+System components inside containers always write to `.log` files in the `/var/log` directory,
+bypassing the default logging mechanism.
+Similar to the container logs, you should rotate system component logs in the `/var/log` directory.
+In Kubernetes clusters created by the `kube-up.sh` script, log rotation is configured by the `logrotate` tool.
+The `logrotate` tool rotates logs daily, or once the log size is greater than 100MB.
+
+## {{% heading "whatsnext" %}}
+
+* Read about the [Kubernetes Logging Architecture](logging.md)
```

Comment:
This hyperlink is incorrect - it should be to the rendered site-relative URL.

Reply:
Can you explain how to fix it?

Reply:
Is the page you want to link to?
nit: Read about the [Kubernetes Logging Architecture](/docs/concepts/cluster-administration/logging/)

Reply:
y